src/main/java/seedu/planner/model/module/ModuleInfo.java

```diff
@@ -1,8 +1,6 @@
 package seedu.planner.model.module;
 
-//@@author GabrielYik @@author rongjiecomputer
-
-import static seedu.planner.commons.util.CollectionUtil.requireAllNonNull;
+//@@author GabrielYik
```

Comment:
Actually you can remove my name haha. This class is essentially yours

Reply:
If I touch this file again next time, I will just own the whole file. Lazy to change this now because I have another PR that needs this merge first.